ynput/ayon-core
# Blender: Model publish uv map name validator

## Changelog Description

Add model publish validator to ensure uv set is named `map1`.

## Additional info

Blender defaults to UV sets named `UVMap` which can be an issue for a Maya-centric pipeline.
Note that the plug-in is disabled in settings by default.

## Testing notes:

1. Enable validator in settings
2. Publish a blender cube
3. It should invalidate and have a working select and repair action
